superchain/Dockerfile

```diff
@@ -47,14 +47,14 @@ RUN yum -y install python3 python3-pip
   && yum clean all && rm -rf /var/cache/yum
 
 # Install Ruby 2.6+
-RUN amazon-linux-extras install ruby2.6                                                                                 \
-  && yum -y install ruby-devel rubygem-rdoc                                                                             \
-  && yum clean all && rm -rf /var/cache/yum                                                                             \
-  && echo 'install: --no-document' > /usr/local/etc/gemrc                                                               \
-  && echo 'update: --no-document' >> /usr/local/etc/gemrc                                                               \
-  && mkdir -p "$GEM_HOME"                                                                                               \
-  && gem install 'bundler:~>1.17.3' 'bundler:~>2.1.4'
-ENV PATH="$GEM_HOME/bin:$GEM_HOME/gems/bin:$PATH"
+#RUN amazon-linux-extras install ruby2.6                                                                                \
+#  && yum -y install ruby-devel rubygem-rdoc                                                                            \
+#  && yum clean all && rm -rf /var/cache/yum                                                                            \
+#  && echo 'install: --no-document' > /usr/local/etc/gemrc                                                              \
+#  && echo 'update: --no-document' >> /usr/local/etc/gemrc                                                              \
+#  && mkdir -p "$GEM_HOME"                                                                                              \
+#  && gem install 'bundler:~>1.17.3' 'bundler:~>2.1.4'
+#ENV PATH="$GEM_HOME/bin:$GEM_HOME/gems/bin:$PATH"
```

Comment:
Can we simply remove this instead of commenting out? GitHub is our memory if we even need to pull this back from the grave (plus it's not so hard, since it's an `amazon-linux-extras` install).

Can you also ensure you amend the README.md file in the `superchain` directory to remove `ruby` (and `gcc` if it so happens to be there) from there?

Reply:
Good points, thanks, all taken care of.